p2p/server.go

```diff
@@ -95,20 +101,24 @@ type Server struct {
 
 	ourHandshake *protoHandshake
 
-	lock    sync.RWMutex // protects running and peers
-	running bool
-	peers   map[discover.NodeID]*Peer
+	lock sync.RWMutex // protects running and peers
```

Comment:
Style: please put the lock directly above the fields it protects without adding a blank line.